client/components/Login.js

```diff
@@ -1,20 +1,31 @@
-import React, { useState, useEffect } from 'react';
+import React, { useState } from 'react';
 import { connect } from 'react-redux';
-import { getUser, getGithubUser } from '../store';
+import { getUser, getGithubUser, createUser } from '../store';
 
 const LoginClass = props => {
   const [email, setEmail] = useState('');
   const [password, setPassword] = useState('');
 
   const saveHandler = ev => {
     ev.preventDefault();
-    props.getUser({ email, password }).then(() => props.history.push('/'));
+    props
+      .getUser({ email, password })
+      .then(() => props.history.push('/'))
```

Comment:
After logging in should redirect to challenges page.
```suggestion
      .then(() => props.history.push('/challenges'))
```